recipes/LibriSpeech/ASR/CTC/train_with_wav2vec.py

```diff
@@ -64,6 +66,11 @@ def compute_forward(self, batch, stage):
         # Compute outputs
         p_tokens = None
         logits = self.modules.ctc_lin(x)
+        # Upsample the inputs if they have been highly downsampled
+        if self.hparams.upsampling:
```

Comment:
Logits are to be upsampled not downsampled. If the downsampling factor is too high ( starting from 3 actually ) it is useful to avoid outputting a character every 60ms